Config/configure_PRONTO.ini

```diff
@@ -30,9 +32,9 @@ filter_column = Coding_status
 key_word = !x:noncoding_variant && !synonymous_variant
 all_col_output = False
 ;Specify the column names need to be transfered to new data file. Split the column names with ','. NB: the column contents must be the same as they are in the sheet.
-columns = Gene_symbol,Change_summary,Ensembl_transcript_ID,Genomic_location,Exon_number,Protein_change_short,Coding_status,Functional_domain,Depth_tumor_DNA,AF_tumor_DNA,Depth_normal_DNA,AF_normal_DNA,TSO500_LocalApp_class
+columns = Gene_symbol,Ensembl_transcript_ID,Exon_number,Protein_change_short,Coding_status,AF_tumor_DNA
 ;Specify the output result file name. Please note that you type should not exceed 31 characters and not contain any of the characters ':' ' \'  '/'  '?'  '*'  '['  or ']':
-output_table = preMTBTable_Vigdis
+output_table = sequence_summary
 
 [FILTER3]
 ;In this filter section, the input data is the first table generated from FILTER1.

```

Comment:
in line 47: 

```ini
columns = Gene_symbol,Genomic_location,Position,DNA_change,cDNA_change,Protein_change_short,Coding_status,Depth_tumor_DAN,AF_tumor_DNA
```

columns `Position` and `Depth_tumor_DAN` do not exist and shouldn't even show up in your reports. What is `Position` and the other one seems to be a typo.

Reply:
in line 71:

```ini
columns = Gene_symbol,Protein_change_(code),Coding_status,Functional_domain,AF_tumor_DNA,CPSR_ACMG_class,Reporting_judgement
```

column `Protein_change_(code)` does not exist. Which column did you want to reference here?

Reply:
will be verified and removed by @xiaoliz0